examples/experimental/export_example.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        # _experimental_otel_exporter=ZipkinExporter(
</pre><p>are otel traces able be logged to the normal logging database? Or are they only exported to this zipkin thing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1346/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Regardless of exporter, they are logged into our own db as normal. If exporter is set, they are also exported there.